scripts/check-hbs-syntax.sh

```
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
+
+# Find all .hbs files in the public/templates directory
+hbs_files=$(find public/templates -name "*.hbs")
```

Shellcheck (0.11.0)

[warning] hbs_files appears unused. Verify use (or export if used externally).

(SC2034)